examples/external-data/tests/utilities.ts

```diff
@@ -19,3 +19,11 @@ export async function closeServer(server: Server): Promise<void> {
 		});
 	});
 }
+
+/**
+ * Returns a promise that resolves after `timeMs`.
+ * @param timeMs - Time in milliseconds to wait.
+ * @internal
+ */
+export const delay = async (timeMs: number): Promise<void> =>
```

Comment:
Nit: should this go in the `example-utils` package (to avoid re-implementing this behavior in too many places)?

Reply:
Maybe. It has a bit of a code smell. My guess is that there are better patterns.

Reply:
IMO example-utils should be small and its members should not be long-lived.  Code duplication is actually a benefit in the examples - customers are best able to consume an example if they can avoid cross referencing between packages.

The current members of the example-utils package are mostly intended to be "temporary" (though many have been there for a long time).  The goal is to experiment with a pattern broadly across multiple examples and then promote that pattern to a real shipping package that customers can use.  migration-tools is one example that finally has its own package, some of the other functionalities might end up in aqueduct, etc.